pymc/sampling_jax.py

```diff
@@ -116,6 +123,7 @@ def sample_numpyro_nuts(
     init_state_batched = jax.tree_map(lambda x: np.repeat(x[None, ...], chains, axis=0), init_state)
 
     logp_fn = get_jaxified_logp(model)
+    fn = model.fastfn(vars_to_sample)
```

Comment:
Can you give a more informative name and/or a comment that explains what the purpose of this `fn` will be?